packages/core/src/oot/actors/Player.S

```diff
@@ -195,3 +195,15 @@ PATCH_START 0x8007b8b8
    nop
   nop
 PATCH_END
+
+/*
+Replaces:
+  addiu   sp, sp, 0x18
+  jr      ra
+   nop
+*/
+PATCH_START 0x80079848
+  lw      a0,0x18(sp)
+  j       Player_AfterSetEquipmentData
+   addiu  sp, sp, 0x18
```

Comment:
No space, shorthand notation for simple adds:

`addiu sp,0x18`

Reply:
What do you mean by no space?

Reply:
After comma